### PR TITLE
c31 translation update

### PR DESCRIPTION
**Pull request by ManlyMarco**
Thanks to @ScrewThisNoise for providing the translations.

### CODE REVIEW

**Review by GeBo1 — CHANGES_REQUESTED** [review]
Regex errors (look for `$#` in the new lines)

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/25/76/translation.txt [hunk 1]
```diff
@@ -4,5 +4,5 @@
 「プールの良い所は汗が目立たないところだな、おまえに嗅がれるのを気にする事もない」=“The good thing about the pool is, sweat really doesn't stand out. I don't have to be worried about you smelling it either.”
 「[H名]って汗っかきだったの？　という事はこのプールには大量の[H名]エキスが……」=“You were sweating, [H名]? That means, this pool has lots of your extract, huh...”
 「ば、ばかっ！　何を考えている！　飲んだりしたら、怒るからな！」=“D-Dummy--! What are you thinking! If you actually try drinking it, I'll get mad, okay!?”
-「はは、流石にそこまではしないよ。……しないよ？」=“Haha, even I won't go that far.I won't, okay?”
+「はは、流石にそこまではしないよ。……しないよ？」=“Haha, even I won't go that far.I$3 $4 won't, okay?”
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/25/76/translation.txt:7]:
```suggestion
「はは、流石にそこまではしないよ。……しないよ？」=“Haha, even I won't go that far.I won't, okay?”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/15/76/translation.txt [hunk 2]
```diff
@@ -4,5 +4,5 @@
 「プールの良い所は汗が目立たないところだな、おまえに嗅がれるのを気にする事もない」=“The good thing about the pool is, sweat really doesn't stand out. I don't have to be worried about you smelling it either.”
 「[H名]って汗っかきだったの？　という事はこのプールには大量の[H名]エキスが……」=“You were sweating, [H名]? That means, this pool has lots of your extract, huh...”
 「ば、ばかっ！　何を考えている！　飲んだりしたら、怒るからな！」=“D-Dummy--! What are you thinking! If you actually try drinking it, I'll get mad, okay!?”
-「はは、流石にそこまではしないよ。……しないよ？」=“Haha, even I won't go that far.I won't, okay?”
+「はは、流石にそこまではしないよ。……しないよ？」=“Haha, even I won't go that far.I$3 $4 won't, okay?”
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/15/76/translation.txt:7]:
```suggestion
「はは、流石にそこまではしないよ。……しないよ？」=“Haha, even I won't go that far. I won't, okay?”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/15/90/translation.txt [hunk 3]
```diff
@@ -12,4 +12,4 @@
 「おまえにしては冴えた意見だな……あ、普段通り呼んでしまった」=“That's quite the apt comment, considering it's you, [P名]... Ah, I slipped.”
 「はは、無理に使う必要はないよ。でも、たまにはこうして呼び合うのもいいかもね」=“Haha, you don't need to force yourself to use it. But, calling each other like this every once in a while might be nice, don't you think?”
 「そうだな……では、もう一度私の事も呼んでくれるか？　今回はそれで最後にしよう」=“You're right... then, will you call me by my nickname one more time too? Let's end it with that.”
-「うん、いいよ。……[Hあだ名]……あはは、やっぱり照れるね」=“Yeah, alright.[Hあだ名]... Ahaha, it really does make me blush.”
+「うん、いいよ。……[Hあだ名]……あはは、やっぱり照れるね」=“Yeah, alright.[$3 $4Hあだ名]... Ahaha, it really does make me blush.”
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/15/90/translation.txt:15]:
```suggestion
「うん、いいよ。……[Hあだ名]……あはは、やっぱり照れるね」=“Yeah, alright.[Hあだ名]... Ahaha, it really does make me blush.”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/25/90/translation.txt [hunk 4]
```diff
@@ -12,4 +12,4 @@
 「おまえにしては冴えた意見だな……あ、普段通り呼んでしまった」=“That's quite the apt comment, considering it's you, [P名]... Ah, I slipped.”
 「はは、無理に使う必要はないよ。でも、たまにはこうして呼び合うのもいいかもね」=“Haha, you don't need to force yourself to use it. But, calling each other like this every once in a while might be nice, don't you think?”
 「そうだな……では、もう一度私の事も呼んでくれるか？　今回はそれで最後にしよう」=“You're right... then, will you call me by my nickname one more time too? Let's end it with that.”
-「うん、いいよ。……[Hあだ名]……あはは、やっぱり照れるね」=“Yeah, alright.[Hあだ名]... Ahaha, it really does make me blush.”
+「うん、いいよ。……[Hあだ名]……あはは、やっぱり照れるね」=“Yeah, alright.[$3 $4Hあだ名]... Ahaha, it really does make me blush.”
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/25/90/translation.txt:15]:
```suggestion
「うん、いいよ。……[Hあだ名]……あはは、やっぱり照れるね」=“Yeah, alright.[Hあだ名]... Ahaha, it really does make me blush.”
```

**Pull request merged**